geps/gep-1426.md

```diff
@@ -53,7 +53,16 @@ This approach is dependent on both the "frontend" role of the Kubernetes `Servic
 
 ### Why Service?
 
-The GAMMA initiative has been working to bring service mesh use-cases to the Gateway API spec, taking the best practices and learnings from mesh implementations and codifying them in a spec. Most mesh users are familiar with using the Kubernetes `Service` resource as the foundation for traffic routing. Generally, this architecture makes perfect sense; unfortunately, `Service` is far too coupled of a resource. It orchestrates IP address allocation, DNS, endpoint collection and propagation, load balancing, etc. For this reason, it **cannot** be the right long-term answer for `parentRef` binding; however, it is the only feasible option that Kubernetes has for mesh implementations today. We expect this to change (indeed, we hope to be a part of that change), but in the interest of developing a spec now, we must once again lean on the `Service` resource. However, we will provide provisions to support additional resources as a `parentRef`.
+The GAMMA initiative has been working to bring service mesh use-cases to the Gateway API spec, taking the best practices and learnings from mesh implementations and codifying them in a spec. Most mesh users are familiar with using the Kubernetes `Service` resource as the foundation for traffic routing. In particular, service meshes take advantage of the IP address management and corresponding DNS functionality provided by `Service`.
+
+Generally, this architecture makes perfect sense; unfortunately, `Service` is far too coupled of a resource. It orchestrates not only IP address allocation and DNS but also endpoint collection and propagation, load balancing, etc. For this reason, it **cannot** be the right long-term answer for `parentRef` binding; however, it is the only feasible option that mesh implementations have today.
+One of the key value propositions of service meshes is that they can be added "on top" to preexisting deployments. These deployments are very likely to already consist of `Service` resources.
+In particular, the ubiquity of `Service` in the Kubernetes ecosystem and the time and effort needed to get support for a new resource into the ecosystem, for example with managed Kubernetes providers, makes waiting for a replacement infeasible.
+
+We expect this to change and indeed, we plan to be a part of that change. However, in the interest of releasing a spec for service meshes in particular now, the `Service` resource must be supported by the graduated GAMMA spec.
```

Comment:
IMO:

It will not **change** per-se. parentRef is a generic attachment point. It will **always** support Service (and Gateway). It may also add support for NewService or many other types. But that will not mean Service is dropped.

Reply:
Made it clear we're not discussing dropping Service